tests/routes/users.spec.js

```diff
@@ -373,4 +374,33 @@ describe('USER AUTHENTICATION', () => {
       expect(res).to.have.status(200);
     });
   });
+
+  describe('Get all Users', () => {
+    before(async () => {
+      await createTestFakeUsers();
```

Comment:
Good job on using faker to generate data in your database to allow you test with. I think it would be good to clear the User table in your before block before running your test, since your test depends on the number of users it meets in the User table ---- line 391. This will avoid errors incase the User table had values before your test

Reply:
There is no need for that because the user whose token is been used to access the endpoint is created at the top level before hook declared first within the test file. Token can't be gotten from the fake users created.